ui/components/component-library/avatar-account/avatar-account.tsx

```diff
@@ -14,6 +14,9 @@ import {
   AvatarAccountProps,
 } from './avatar-account.types';
 
+/**
+ * @deprecated Use PreferredAvatar or AvatarAccount from the design system
+ */
```

Comment:
non-blocking: Because `PreferredAvatar` includes the logic to set the variant based on user settings we might just be able to suggest that one moving forward. I can't think of a use case to use the AvatarAccount from the design system react library but you might have seen some so there could be!